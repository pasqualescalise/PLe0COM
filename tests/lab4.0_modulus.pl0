VAR i : int;

BEGIN
    print 20 % 16;
	for i = 1; i < 35; i = i + 1 do begin
		print 35 % i
	end
END
