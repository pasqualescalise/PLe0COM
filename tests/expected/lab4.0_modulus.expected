4
0
1
2
3
0
5
0
3
8
5
2
11
9
7
5
3
1
17
16
15
14
13
12
11
10
9
8
7
6
5
4
3
2
1
0
